### DISCUSSION TITLE
Do I manually need to add the keyPrefix when using .get() ?

### DISCUSSION THREAD

**Discussion by The-LukeZ** [animir/node-rate-limiter-flexible#272] [General]
When I have a ratelimiter with a keyPrefix `whl` - do I have to manually add this keyPrefix to the key?

Example:
```js
const limiter = new RateLimiterMemory({
    keyPrefix: "whl",
    points: 1,
    duration: 60,
    blockDuration: 60,
});

// without "whl"
limiter.get(user.id)
// or with "whl"
limiter.get("whl" + user.id)
```

**Comment by animir** [comment]
@The-LukeZ Hi, you don't have to add prefixes manually.
It is handled under the hood for all functions.

Btw, if you want to know internal ket with prefix you can call [getKey](https://github.com/animir/node-rate-limiter-flexible/wiki/API-methods#ratelimitergetkeykey) method.
